lib/vanagon/platform/windows.rb

```diff
@@ -8,7 +8,10 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         ["mkdir -p output/#{target_dir}",
-        "touch output/#{target_dir}/#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"]
+        "mkdir -p #{project.name}",
+        "cp #{project.name}.nuspec #{project.name}/",
+        "(cd #{project.name}; /cygdrive/c/ProgramData/chocolatey/choco pack)",
```

Comment:
I'm not sure if we need anything that complicated. We might get away with having a method that returns the drive root, in this case `/cygdrive/c`. In the future we could change that as needed.